scene/gui/popup.cpp

```diff
@@ -223,6 +223,8 @@ Popup::Popup() {
 	set_transient(true);
 	set_flag(FLAG_BORDERLESS, true);
 	set_flag(FLAG_RESIZE_DISABLED, true);
+	set_flag(FLAG_MINIMIZE_DISABLED, true);
+	set_flag(FLAG_MAXIMIZE_DISABLED, true);
```

Comment:
Popups are borderless.
Did you mean it for dialogs?

Reply:
Dialogs are resizable, but maximizing and minimizing them is not useful, so it's probably ok to set it as default for `AcceptDialog`. It can be overridden for specific dialogs if it is actually desired.